hacker-news-app/src/Components/LogInPage/loginStyles.css

```diff
@@ -0,0 +1,43 @@
+.username, .password {
+    width: 75%;
+    max-width: 300px;
+    border: none;
+    border-bottom: 2px solid #F2D6EB;
+    margin: 20px auto;
+    padding: 5px;
+    font-size: 1rem;
+}
+
+.remember_password {
+    margin: 10px;
+}
+
+.btn {
+    color: white;
+    font-size: 0.8rem;
+    background-color: #470938;
+    padding: 8px;
+    margin: 10px auto;
+    font-weight: bold;
+    border-radius: 5px;
+}
+
+.btn:hover{
+    color: #470938;
+    background-color: white;
+    border: 1px solid #470938;;
+}
+
+.errorMessage{
+    color: red;
+    font-size: 0.8rem;
+    font-style: italic;
+    position: relative;
+    bottom: 15px;
+}
+
+
+.communityLinks{
+    color: white;
+    text-decoration: none;
+}
```

Comment:
Again, just to be consistent I'd suggest to write styles in components with the help of styled-components

Reply:
All sorted